app/javascripts/app.js

```diff
@@ -144,7 +150,14 @@ async function runAt(address) {
 
 function load() {
   console.log("window.ethereum = ", window.ethereum);
-  web3c = new Web3c(window.ethereum);
+  if (getUrlParameter('insecureTestingKeys') === '1') {
+    console.warn("Using unsecret key manager signing key");
+    web3c = new Web3c(window.ethereum, undefined, {
+      keyManagerPublicKey: '0x9d41a874b80e39a40c9644e964f0e4f967100c91654bfd7666435fe906af060f',
```

Comment:
do we need this one specifically, or you you set it up as
`secretballot.html?keymanagerKey=0x....` where it can be specified in the URL?

Reply:
we need this one specifically. don't need other keys at this time

Reply:
it seems less hacky to have the person who is running the custom network also specify this key rather than hardcoding it in the app?

Reply:
yeah, but it's also an unnecessarily powerful flow of untrusted input into the app. currently `=== '1'` has the benefit that validation is simple

Reply:
Additional info: this public key corresponds to an insecure key used for local key manager testing. cc @peterjgilbert